contracts/v0.6.4/LibraryPerson.sol

```diff
@@ -0,0 +1,48 @@
+pragma solidity ^0.6.4;
```

Comment:
can we move these contracts to a separate PR? lets just fix windows development in this one

Reply:
Absolutely, sorry I didn't realize it had taken multiple commits.

Reply:
Will you find time to fix it or should I step in? :)

Reply:
Absolutely, I just removed the files, sorry again, I was convinced I was committing to my fork.
The PR idea was just for the "compile-contract.ts" file.
Contributing.md are my personal notes, do we keep them or do you prefer to remove them?

Thanks

Reply:
Thanks!